applications/TradeCoin.md

```diff
@@ -0,0 +1,147 @@
+
+We take inspiration from the latest trend in the blockchain ecosystem which we aim to replicate on polkadot.
+
+Unibot is a Telegram bot that allows users to trade cryptocurrencies without leaving the Telegram app. Unibot offers various features, such as decentralized copy trading, limit orders on decentralized exchanges (DEXs), protection against MEV (Miner Extractable Value) bots, and more. The platform has gained momentum and has shown significant growth in trade volume, revenue, and user counts over the last two months. Unibot also has its native token called UNIBOT, with holders receiving a portion of transaction fees and trading volume.
+
+The UNIBOT has seen substantial user growth and has facilitated trades worth $54 million, distributing nearly $1 million in revenue back. Holders of UNIBOT tokens receive rewards of 40% of transaction fees and 1% of the platform's total trading volume.
+
+The appeal of Unibot and other Telegram-based trading bots lies in their user-friendly interface, providing ease of use compared to decentralized exchanges like Uniswap. With Unibot, traders can execute market buy/sell orders through a simple Telegram command, eliminating the need for continuous wallet logins and cross-checking token information. This streamlined process is particularly advantageous for fast swaps and scalping strategies, attracting traders who value efficiency and.
+
+Unibot's success has received endorsement from the crypto community, with popular crypto traders on Twitter praising the platform's user experience and its advantage of trading directly through Telegram for various trading strategies. As a result, newer tokens like WagieBot, 0xSniper, Bridge, and Bolt have experienced significant growth, emulating Unibot's trajectory.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Alexandr Fedotov
+
+### Contact
+
+- **Contact Name:** Alexandr Fedotov
+- **Contact Email:** alexkekfedotov@gmail.com
+- **Website:** trade-coin-xi.vercel.app/
+
+### Legal Structure
+
+Individual
+
+### Team's experience
+
+Our project won in XDC hackathon. Previously, its name was "XRC-Swap" 
+https://xdc.devpost.com/project-gallery
+
+Also, our project was a finalist on "TRON DAO". This thread has had a lot of comments and positive feedback.
+
+https://forum.trondao.org/t/overview-of-the-tradecoin-project/16511
+
+The project also participated in the "FVM Impact Evaluator". Follow the link you can find us:
+
+https://spacewarp.fvm.dev/#ie
+
+### Team Code Repos
+
+- https://github.com/AlexFedotovqq/TradeCoin
+
+- https://github.com/AlexFedotovqq
+
+### Team LinkedIn Profiles (if available)
+
+No
+
+## Development Status :open_book:
+
+I have posted updates on [tron dao thread](https://forum.trondao.org/t/overview-of-the-tradecoin-project/16511)
+
+And I have also created new figma wireframes for a DeFi interface: https://www.figma.com/file/rhOpmDzGZHYr4RoIIgInjj/TradeCoin%5C?type=design&node-id=6%3A353&mode=design&t=qn3JKAKV7rDls0Dr-1
```

Comment:
The grant application is for a telegram bot, and these wireframes show a website. How are they related?